packages/terra-graphs-docs/src/terra-dev-site/doc/Graphs/Combination/BasicCombinationGraph/LineWithMultipleBubbleData.5.doc.mdx

```diff
@@ -1,30 +1,25 @@
-import CombinationGraph from '../../../example/Graphs/CombinationGraph/BasicCombinationGraphs/LineWithMultipleBubbleData?dev-site-example';
-import PropsTable from '@cerner/terra-graphs/lib/components/Combination/CombinationGraph?dev-site-props-table';
+import CombinationGraph from '../../../example/Graphs-Carbon/CombinationGraph/BasicCombinationGraphs/LineWithMultipleBubbleData?dev-site-example';
 import GraphConfig from '@cerner/terra-graphs-docs/lib/example-datasets/graphConfigObjects/Combination/combinationDefault.js?dev-site-codeblock';
 import DataObject from '@cerner/terra-graphs-docs/lib/example-datasets/dataObjects/Combination/lineWithMultipleBubbleData.js?dev-site-codeblock';
 require('details-polyfill')
 
 # Line With Multiple Bubble dataset
 This is a simple Line Graph combined with multiple Bubble dataset.
 
-#### Note: "graphType" should be added for each dataset in combination graph.
-
 ## Getting Started
 
 - Install with [npmjs](https://www.npmjs.com):
-  - `npm install terra-graphs`
+  - `npm i @cerner/carbon-graphs --save-dev`
 
 ## Usage
-```jsx
-import React from 'react';
-import CombinationGraph from '@cerner/terra-graphs/lib/components/Combination/CombinationGraph';
+```js
+import Carbon from "@cerner/carbon-graphs/dist/js/carbon-graphs.js";
+import "@cerner/carbon-graphs/dist/css/carbon-graphs.css";
```

Comment:
This and all the other places.

```suggestion
import Carbon from '@cerner/carbon-graphs/dist/js/carbon-graphs.js';
import '@cerner/carbon-graphs/dist/css/carbon-graphs.css';
```

Reply:
Fixed in [this](https://github.com/cerner/terra-graphs/pull/181/commits/9eb4a3d4b7f59e64d194815b5394d3e8d8d95ba8) commit.